docs/guides/validators/security_best_practices.md

```diff
@@ -447,9 +460,11 @@ sudo iptables -A INPUT -p tcp --dport 26657 -j DROP
 sudo iptables -A INPUT -p tcp --dport 26658 -j DROP
 ```
 
-`Port` 26657 is using by default for RPC server and if you want you can limit incoming connections too.
+::: tip
+
+If you want to use haqq node CLI remotely you might change IP address `laddr` to 0.0.0.0
```

Comment:
Let's stick to just minimum and necessary.
We should give an instruction on how to secure validator on firewall level. This one is confusing and can cause operators to expose something which should not exposed.

Reply:
replaced